QUEST_LV_0100.tsv

```diff
@@ -8606,58 +8606,58 @@ QUEST_LV_0100_20151001_008605	$Since we were able to sealed the Kartas Hunter he
 QUEST_LV_0100_20151001_008619	$Place the Corrupt Fangs near the seal. {nl}Then, our last mission will start.
 QUEST_LV_0100_20151001_008620	$Everything is over...{nl}It is so fortunate that it ended even after our deaths
 QUEST_LV_0100_20151001_008621	$Thank you again. Please pass this crystal to my daughter's friend.{nl}Now, I will return to the goddesses together with my vassals.{nl}
-QUEST_LV_0100_20151001_008622	$If you meet my daughter, please tell her that I am really sorry.{nl}And that I love her... I hope you can pass my words to her.
-QUEST_LV_0100_20151001_008623	$I wonder who's the girl who Legwyn tried to protect until his death?{nl}Nevertheless, I am sure that he will rest in peace. Since his last mission has been fulfilled.{nl}
+QUEST_LV_0100_20151001_008622	$If you meet my daughter, please tell her that I'm really sorry and that I love her...{nl}I hope you can pass my words to her.
+QUEST_LV_0100_20151001_008623	$I wonder who's the girl who Legwyn tried to protect until his death?{nl}Nevertheless, I'm sure he'll rest in peace since his last mission has been fulfilled.{nl}
 QUEST_LV_0100_20151001_008624	$I will return this to Erikas now.{nl}I really appreciate your help on behalf of my friend, Justina Legwyn.
-QUEST_LV_0100_20151001_008625	I can tell this to you freely.{nl}The keepsakes of Lydia Schaffen... I came out from the Astral Tower with my colleagues to protect them.{nl}
-QUEST_LV_0100_20151001_008626	The predecessor leader died suspiciously and Ignas became the next leader of the Astral Tower.{nl}And then his tyranny started.{nl}
-QUEST_LV_0100_20151001_008627	He told everyone that the old rules of Lydia Schaffen should be abandoned and everything should start again...{nl}He told us to eliminate everything that was related to Lydia Schaffen.{nl}
-QUEST_LV_0100_20151001_008628	But, Lydia Schaffen was the everything of the Astral Tower and she was the symbolic figure of all archers.{nl}After we regained consciousness, we ran away with our colleagues and the keepsakes.{nl}
-QUEST_LV_0100_20151001_008629	Right now... I can hardly move due to the injury.{nl}But, if I stay like this, the chasers would find out about it.
-QUEST_LV_0100_20151001_008630	I think we should move.{nl}Can you lure the chasers so that I could go to Guoda Empty Lot?{nl}
-QUEST_LV_0100_20151001_008631	When we fire this disposable signal bomb, they will follow you.{nl}I am counting on you...
+QUEST_LV_0100_20151001_008625	I can tell you this at least.{nl}The keepsakes of Lydia Schaffen...{nl}I came out from the Astral Tower to protect them with my colleagues.{nl}
+QUEST_LV_0100_20151001_008626	The preceding leader suspiciously died and Ignas became the next leader of the Astral Tower.{nl}Then, his tyranny started.{nl}
+QUEST_LV_0100_20151001_008627	He told everyone that the old rules of Lydia Schaffen should be abandoned and that we should start over...{nl}He said to eliminate everything that was related to Lydia Schaffen.{nl}
```

Comment:
He told everyone that the old rules of Lydia Schaffen were to be abandoned and that...

or

He told everyone to abandon the old rules of Lydia Schaffen, and to start over...